packages/web-app-files/src/components/AppBar.vue

```diff
@@ -182,6 +183,8 @@ export default {
       }
     },
     canUpload() {
+      // return this.currentFolder.permissions.indexOf('C') >= 0
+      // borrowed from packages/web-app-files/src/helpers/resources.js
```

Comment:
@kulmann could we have a look into this function? I'll revert the commit once we found a solution and commit that

Reply:
For the record, we talked about this in a call. `canUpload` already does what it should. The `v-oc-tooltip` directive has a bug.